backend/src/test/java/org/cryptomator/hub/api/VaultResourceTest.java

```diff
@@ -843,6 +843,25 @@ public void testCreateVault2() {
 					.then().statusCode(400);
 		}
 
+		@Test
+		@DisplayName("PUT /vaults/7E57C0DE-0000-4000-8000-0001FFFFAAAA returns 201")
```

Comment:
Can we add some more text here? E.g. `...returns 201 ignoring archived flag`

Reply:
Fixed in 0df8420076f892e01e91ddb372e795ac40ee0c63.